features/git-hack/hack_push_strategy_local.feature

```diff
@@ -0,0 +1,31 @@
+Feature: git hack: don't push branch to remote upon creation
+
+  As a developer starting work on a new private feature
+  I don't want git-hack to push my branch to the remote repo
+  So that the upstream repo is not aware of my work
```

Comment:
Is thin really the motivation for this feature? What about `So that my CI server doesn't unnecessarily run tests for commits without changes`?